src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -113,6 +118,20 @@ protected function createSeeder()
         ]);
     }
 
+    /**
+     * Create a policy file for the model.
+     *
+     * @return void
+     */
+    protected function createPolicy()
+    {
+        $policy = Str::studly(class_basename($this->argument('name')));
+
+        $this->call('make:policy', [
+            'name' => "{$policy}Policy",
```

Comment:
Wouldn't you want to pass `--model` here as well, that way the policy is generated for the model

Reply:
I had debated that myself.  Left it off as I felt that a quick scaffold would be better than nothing. But, if the community thinks that this is an option that would be better added in, it's something I can include quick and easy!

Reply:
If we were to ever include this it would definitely need to pass `--model` to `make:policy`.

Reply:
Done!